ui/apps/platform/src/Containers/Policies/Wizard/Step3/PolicyCriteriaFieldInput.tsx

```diff
@@ -131,6 +131,7 @@ function PolicyCriteriaFieldInput({
                         isDisabled={readOnly}
                         selections={value.value}
                         placeholderText={descriptor.placeholder || 'Select an option'}
+                        menuAppendTo={() => document.body}
```

Comment:
👍 This is something that probably needs to be applied in quite a few places. (RE: Violations table menu issue that Boaz just brought up.)

Reply:
@stackrox/ui 
This is a good little task for the "99 paper cuts" in January/February that Michael Hess will be announcing.